mllib/src/main/scala/org/apache/spark/ml/tuning/TrainValidationSplit.scala

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.spark.ml.tuning
 
+import java.{util => ju}
```

Comment:
I think its more common in our current code to do:
`import java.util.{List => JList}`